admin/lib/validator/check-form/single-check-form-validator.js

```diff
@@ -40,7 +40,7 @@ singleCheckFormValidator.validate = async (uploadedFile) => {
   const fileContent = fileBuffer && fileBuffer.toString().trim()
   const fileLines = fileContent && fileContent.split('\n').length
   // Invalid total file lines
-  if (fileLines !== config.LINES_PER_CHECK_FORM) {
+  if (fileLines !== parseInt(config.LINES_PER_CHECK_FORM)) {
```

Comment:
can you please parse the value on a separate line, and deal with the fact that it could return `NaN`